src/pip/_internal/configuration.py

```diff
@@ -360,6 +360,8 @@ def _normalized_keys(self, section, items):
     def _get_environ_vars(self):
         # type: () -> Iterable[Tuple[str, str]]
         """Returns a generator with all environmental vars with prefix PIP_"""
+        if 'VIRTUAL_ENV' in os.environ:
```

Comment:
We should use `pip._internal.utils.virtualenv.running_under_virtualenv` instead - as-is this will break if executing something like `/usr/bin/python -m pip ...` explicitly.

Reply:
I've pushed that change as https://github.com/pypa/pip/pull/6984/commits/5802ff682562950ae30e9253ff756aeedfd43b31